test/integration_test.py

```diff
@@ -487,6 +491,19 @@ def run(thread_count):
         # Reset to pre-test state.
         portal_service.overwrite(old_db)
 
+    def _test_check_all_indices(self):
+        """
+        This test serves two purposes. First, by calling ESClientFActory.get(),
+        it validates ES client configuration. Second, it tests the existence of
+        all of the expected indices.
+        """
+
+        log.info("Confirming valid configuration of ESClientFactory ...")
```

Comment:
Remove. There is no point in logging this. It doesn't take long and if it fails it is clear what's going on.